service-admin/composer.json

```diff
@@ -67,7 +61,9 @@
         "phpunit/phpunit": "^9.5.10",
         "roave/security-advisories": "dev-master",
         "squizlabs/php_codesniffer": "^3.6.1",
-        "vimeo/psalm": "^4.23"
+        "vimeo/psalm": "^4.23",
```

Comment:
Are the following two packages in the dev section? If they are, they should be moved into the main (runtime) section as I believe they are runtime dependencies.

Reply:
These packages turn out not to be needed in either section, so now removed